rainier-core/src/main/scala/com/stripe/rainier/compute/Show.scala

```diff
@@ -0,0 +1,38 @@
+package com.stripe.rainier.compute
+
+/**2
+  * Trait for serializable and deserializable types
+  */
+trait Show[-T, S] {
```

Comment:
This name is commonly used for printing to String, so using it for serialization/deserialization seems confusing IMHO.